cinder/volume/rpcapi.py

```diff
@@ -248,7 +248,8 @@ def extend_volume(self, ctxt, volume, new_size, reservations):
         cctxt.cast(ctxt, 'extend_volume', volume=volume, new_size=new_size,
                    reservations=reservations)
 
-    def migrate_volume(self, ctxt, volume, dest_backend, force_host_copy):
+    def migrate_volume(self, ctxt, volume, dest_backend, force_host_copy,
+                       call=False):
```

Comment:
I'd rename that parameter to `wait` or `wait_for_finish` or something. I see where the name comes from, but it's not obvious from the function head. The meaning I first thought it would have, was weather to actually make the call or not. That was a little confusing and renaming should help.

Reply:
Just wanted to hear that from someone else than me.
Then we have enough votes to rename it. I'd say either `wait` or `wait_for_completion`.

Reply:
Let's do `wait_for_completion` then.